CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Oneview Features supported
 - Enclosure
 - Enclosure Group
+- Hypervisor Cluster Profile
```

Comment:
Line number 6 says Hypervisor Manager and Server Certificate resources are supported. But it's not listed below. Task resource doesn't exist. Remove this. Fix theCHANGELOG

Reply:
Done

Reply:
You don't need to add these entries manually. rebase you branch with the master.